nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -20,6 +20,7 @@
 
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
+import java.time.temporal.Temporal;
```

Comment:
`java.time` is from Java 8 but we must support Java 7 for nuxeo-java-client, so you should remove this (I guess it's ok in tests though).